sdk/metricsadvisor/ai-metrics-advisor/review/ai-metrics-advisor.api.md

```diff
@@ -190,9 +190,69 @@ export type ChangeThresholdConditionUnion = {
     suppressCondition: SuppressCondition;
 };
 
+// @public
+export type CreateAnomalyAlertConfigurationResponse = CreatedAnomalyAlertConfiguration & {
+    _response: coreHttp.HttpResponse & {
+        parsedHeaders: any;
+    };
+};
+
+// @public
+export type CreateAnomalyDetectionConfigurationResponse = CreatedAnomalyDetectionConfiguration & {
+    _response: coreHttp.HttpResponse & {
+        parsedHeaders: any;
+    };
+};
+
+// @public
+export interface CreatedAnomalyAlertConfiguration {
+    id: string;
+}
+
+// @public
+export type CreatedAnomalyDetectionConfiguration = {
+    id: string;
+};
+
 // @public
 export type CreateDataFeedOptions = DataFeedOptions & OperationOptions;
 
+// @public
+export type CreateDataFeedResponse = CreatedDataFeed & {
+    _response: coreHttp.HttpResponse & {
+        parsedHeaders: any;
+    };
+};
+
+// @public
+export type CreatedDataFeed = {
```

Comment:
@bterlson is it a concern to add more types, or should we inline `id` in the `Create.*Response` types?